.github/workflows/python-package.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish Python 🐍 distributions 📦 to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI and TestPyPI
```

Comment:
slick, I love this workflow :)

Reply:
me too! Pretty darn cool. The use of git tags is so nice.